### PR TITLE
created a workaround for missing data fields, added comments

### PR DESCRIPTION
**Pull request by jtrejox**
I made of few fixes and workarounds for some missing data fields in the NYT data, I wasn't able to work on the local storage part today but I will work on it tomorrow, I also added comments to the script.js file and added tags so that you can find where I am creating new elements and apply the css there.

If you are in the script.js file just do a search(ctrl+s) for CSS all in caps and you will see the tags, hope it helps, also hope I didn't break any of the css you already added, if I did then let me know and I can help you sort it out

There where some missing fields in the NYT data so I had to add some elements to the modal, you will see them in the HTML,
basically some of the older articles in the NYT database don't have News Desk so there is no way to sort them, I added an option to see the articles anyway as a giant list.

To see what I am talking about, search for the movie "top gun" then click on the Tom Cruise movie, you should see the layout I  showed you in class.
Then try clicking on the cowboy movie "Top Gun" and you should see a single "Articles" button and a disclaimer saying the articles can't be organized
It's all in the HTML I'm just using .js to hide and unhide the elements, hopefully that makes it easier to add css

### CODE REVIEW

**Comment by Jr-source** [comment]
Thanks Jorge. I will look at it tomorrow as well and follow up on any issues I have.
